src/main/java/edu/kis/powp/jobs2d/factories/TriangleFactory.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.factories;
+
+import edu.kis.powp.jobs2d.drivers.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.command.OperateToCommand;
+import edu.kis.powp.jobs2d.drivers.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.drivers.command.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class TriangleFactory {
+
+    public static ComplexCommand createCircle(DriverManager driverManager) {
```

Comment:
not very precise circle (or triangle) ;)
Passing a manager instead of a driver increases dependencies.